packages/gatsby/src/internal-plugins/query-runner/query-compiler.js

```diff
@@ -104,6 +104,7 @@ class Runner {
     let files = [
       path.join(this.base, `src`),
       path.join(this.base, `.cache`, `fragments`),
+      path.join(this.base, `node_modules`, `gatsby-seo`),
```

Comment:
We will need this to avoid `Loading StaticQuery...` issues.

Should I conditionally check whether this directory exists? It will just ignore it as far as I know if it's empty--so not a huge concern.

Reply:
I really don't like targeting specific packages in gatsby core to extract queries. This is something that ultimately will need to be solved to enable query extraction (and nicer fragments extraction as what plugins currently do to register fragments is copying files with fragment definitions to arbitrary `.cache/fragments` location)

Reply:
Totally agree. I'll think on a better way to do this.